### ISSUE TITLE
[QUESTION]: Saving the output logs of experiments

### ISSUE THREAD

**Issue by jamshaidsohail5** [chemprop/chemprop#399]
Hi. I hope this message finds you in the best of health. I am using chemprop for uncertainty quantification using the below repostory, which is built on top of chemprop. My question is how we can save the full logs of the experiments in chemprop in a file. For example I want to have a look how validation rmse is decreasing over the epochs and what is the loss of one epoch ? Does chemprop provides any option while running the experiments on how to save the logs ?

https://github.com/lhirschfeld/ChempropUncertaintyQuantification/tree/master

**Comment by kevingreenman** [comment]
The main version of Chemprop (this repo) saves logs (`verbose.log` and `quiet.log`) by default. We're not able to help with forks of Chemprop since we don't maintain those repos ourselves. Have you considered using the [uncertainty estimation tools](https://github.com/chemprop/chemprop#uncertainty-estimation) built into this repo rather than those in the fork?

**Comment by cjmcgill** [comment]
Seconding @kevingreenman , many of the most promising tools from the other repo have been implemented in main chemprop now.

The loss values through training can also be viewed using tensorboard to view the events.out.tfevents file that is generated alongside the model.pt file when models are trained. It's the same information you can view in the log files, but more readily viewable.

The training loss is logged every 10 batches by default. There's not currently a way to log it for whole epochs. The training loss for regression models is a mean-squared-error of a scaled target. The validation rmse is calculated at the end of every epoch on unscaled targets. Because of the MSE/RMSE difference, the batch/epoch difference, and the scaled/unscaled difference these values aren't directly comparable but can still be useful for judging trends versus each other.
